vrep/lbr4-torque-control/lbr4_torque_control.m

```diff
@@ -44,7 +48,7 @@
 %% Main control loop for the first desired pose
```

Comment:
@ffasilva please rewrite the comment here. It seems there is no control loop or desired pose in the example. This could be potentially confusing for some users.

Reply:
Done in [6efb2dd](https://github.com/dqrobotics/matlab-examples/pull/8/commits/6efb2ddf995be92226260838540a2b3b5303b43e).